From: Andreas Henriksson <andreas@fatal.se>
Date: Tue, 10 Sep 2019 17:14:15 +0200
Subject: Add fallback for when tree_index.sgml is not available

Newer versions of gtk-doc-tools will not generate empty files anymore.
https://gitlab.gnome.org/GNOME/gtk-doc/commit/97541700fe55bf5ba1522773dd242a4598cac187
---
 doc/rsvg-docs.xml | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/doc/rsvg-docs.xml b/doc/rsvg-docs.xml
index 40c056a..8b4889a 100644
--- a/doc/rsvg-docs.xml
+++ b/doc/rsvg-docs.xml
@@ -98,7 +98,13 @@
 
   <chapter xml:id="object-tree">
     <title>Object Hierarchy</title>
-     <xi:include href="xml/tree_index.sgml"/>
+     <xi:include href="xml/tree_index.sgml">
+       <xi:fallback>
+         <para>
+           not available.
+         </para>
+       </xi:fallback>
+     </xi:include>
   </chapter>
   <index xml:id="api-index-full">
     <title>API Index</title>
